test/integration/parachain/staging/sequential/replace.test.ts

```diff
@@ -155,9 +180,8 @@ describe("replace", () => {
                 // make sure vault does not hold enough issued tokens to request a replace
                 const replaceAmount = dustValue.add(tokensInVault);
 
-                const replacePromise = callWith(interBtcAPI, vault_2, () =>
-                    interBtcAPI.replace.request(replaceAmount, collateralCurrency)
-                );
+                interBtcAPI.setAccount(vault_2);
```

Comment:
I believe we can move these introduced `setAccounts` outside the loop. Not a huge boost, though, so not critical imo.